ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_extrema_finding.py

```diff
@@ -57,6 +57,45 @@ def test_numpy_minimum(
         subok=True,
     )
 
+# fmin
+@handle_frontend_test(
+    fn_tree="numpy.fmin",
+    dtype_x1_x2_axis=helpers.dtype_values_axis2(
```

Comment:
This seems to be an error right?
```suggestion
    dtype_x1_x2_axis=helpers.dtype_values_axis(
```